az-digital/az_quickstart
# WIP Closes #2693 Add media_remote and Tableau Visualization, Google Calendar support.

## Description

Adding remote media embed capability

## Related issues
#2693

## How to test
See embed examples: /pages/media-embeds

Go to /media/add
See the Google Calendar option
Read description
Click link to add Google calendar
Find a Google Calendar to embed
Find the embed URL for that calendar
Paste it into the  Google Calendar URL field
See the new calendar in the media list in `/admin/content/media`


Alternative approach:
Create a page
Add a text paragraph
In the CKEditor click the add Media icon
Choose Google Calendar from the vertical tabs
Add your Google Calendar URL in the provided field save and embed (Alternatively choose the google calendar you added in the previous steps.)


Navigate to your page and see if the calendar appears

Navigate to /pages/media-embeds to see the demo content provided by this PR.

The same steps can be followed to add a Tableau Visualization

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
